client/pk-offline-update.c

```diff
@@ -150,15 +150,19 @@ pk_offline_update_progress_cb (PkProgress *progress,
 			return;
 		sd_journal_print (LOG_INFO, "percentage %i%%", percentage);
 
+		g_autofree gchar *tmp_perc = NULL;
```

Comment:
This declaration needs to be moved to the top, else you'll get `mixed declarations and code` warning from the compiler.

Reply:
Done.